### PR TITLE
11300-radius-not-work-similar-estate

### PR DESCRIPTION
**Pull request by hungnc89**

### CODE REVIEW

**Comment by jayay** [comment]
Looks good. Any change this can be tested?

File: plugin/EstateDetail.php [hunk 1]
```diff
@@ -144,16 +151,25 @@ public function getMovieEmbedPlayers(array $options = []): array
...
+	 * @throws API\APIEmptyResultException
+	 * @throws API\ApiClientException
+	 * @throws DataView\UnknownViewException
+	 * @throws DependencyException
+	 * @throws NotFoundException
+	 * @throws HttpFetchNoResultException
 	 */
 
-	public function getSimilarEstates(): string
+	public function getSimilarEstates($pContainer): string
```
Comment (jayay) [inline plugin/EstateDetail.php:164]: This would break since it is being called in the `default_detail` template.

**Review by jayay — CHANGES_REQUESTED** [review]
Changing the signature of `\onOffice\WPlugin\EstateDetail::getSimilarEstates` might break the templates

File: plugin/EstateDetail.php (hunk 1 above)
Comment (jayay) [reply plugin/EstateDetail.php:164]: It could be an optional parameter

**Comment by jayay** [comment]
Is there any news on this?

**Comment by nglelinh** [comment]
The discussion about this PR is here https://github.com/orgs/onOfficeGmbH/teams/oo-wp/discussions/16
We mentioned that with current structure we could not write unittest for the feature. @jayay  Could you please consider again?

**Review by jayay — APPROVED** [review]

**Pull request merged**